charts/kong/values.yaml

```diff
@@ -590,7 +596,7 @@ enterprise:
     smtp_admin_emails: none@example.com
     smtp_host: smtp.example.com
     smtp_port: 587
-    smtp_auth_type: ''
+    smtp_auth_type: nil
```

Comment:
If nothing else, this change should be removed. It accidentally reverts https://github.com/Kong/charts/commit/c66fa64dc4944252ff650d042507d08fe24152e9, which we don't want.